avm/res/managed-identity/user-assigned-identity/main.bicep

```
@@ -0,0 +1,145 @@
+metadata name = 'User Assigned Identities'
+metadata description = 'This module deploys a User Assigned Identity.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Optional. Name of the User Assigned Identity.')
+param name string = guid(resourceGroup().id)
```

```suggestion
@description('Required. Name of the User Assigned Identity.')
param name string
```

@eriqua would you agree we should not have this default value?

---

Note: This was already implemented like this (i.e., with the default value in CARML)

---

Agree, sorry I didn't catch this and the other 2 points below. Thanks!